buildSrc/settings.gradle.kts

```diff
@@ -67,3 +67,19 @@ for (project in rootProject.children) {
     assert(project.buildFile.isFile)
 }
 
+fun remoteBuildCacheEnabled(settings: Settings) = settings.buildCache.remote?.isEnabled == true
+
+fun getBuildJavaVendor() = System.getProperty("java.vendor")
+
+fun getBuildJavaHome() = System.getProperty("java.home")
+
+gradle.settingsEvaluated {
+    if (remoteBuildCacheEnabled(this) && !getBuildJavaVendor().contains("Oracle") && !JavaVersion.current().isJava9()) {
```

Comment:
I think we should disable this check, too, if the system property is set. This way we can start trying out different versions of Java on CI and locally.

Reply:
How about now? If the property is present, ignore the following check completely.

Reply:
Looks good now.